hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java

```diff
@@ -336,6 +338,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
 
     switch (result) {
     case SUCCESS:
+      omMetrics.incNumKeys(numMissingParents);
```

Comment:
Should this be numMissingParents +1 ?

Reply:
The metric being incremented here is because of the missing directories. The key/file value is incremented because of metric increment in OmKeyCommitRequest

Reply:
Thanks for the detail. Makes sense to me. Can we add some comments on this?